README.rst

```diff
@@ -102,14 +102,14 @@ Overview of Features
 --------------------
 
 * Extensive gallery of examples (see `Quick Examples`_)
-* Embeddable rendering in Jupyter Notebooks (static and interactive with `Panel`_)
+* Embeddable rendering in Jupyter Notebooks (static and interactive with `ipyvtk_simple`_)
```

Comment:
```suggestion
* Interactive plotting in Jupyter Notebooks (server-side rendering with `ipyvtk_simple`_)
```

Just a tweak to make sure we are conveying this correctly (it is not static/embeddable since it is server-side and requires an open Python thread.